src/strategies/gnosis/swapr/strategy-swapr-$cow-$gno-lp.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "../strategy-swapr-base.sol";
+
+contract StrategySwaprCowGnoLp is StrategySwaprFarmBase {
```

Comment:
Special characters such as `$` are considered "illegal" in file names. Please remove all of them